src/Admin/BaseFieldDescription.php

```diff
@@ -132,6 +132,25 @@ abstract class BaseFieldDescription implements FieldDescriptionInterface
      */
     private static $fieldGetters = [];
 
+    /**
+     * NEXT_MAJOR: Remove the null default value and restrict param type to `string`.
+     */
+    public function __construct(?string $name = null, array $options = [])
+    {
+        // NEXT_MAJOR: Remove this check and keep the else part.
+        if (null === $name) {
+            @trigger_error(sprintf(
+                'Passing no argument to %s() is deprecated since sonata-project/admin-bundle 3.x.'
+                .' In 4.0 the field description name will be required.',
```

Comment:
```suggestion
                'Omitting the argument 1 for "%s()" is deprecated since sonata-project/admin-bundle 3.x.'
                .' It will be mandatory in version 4.0.',
```

Reply:
Changed